packages/typescript/internal/ts_project.bzl

```diff
@@ -618,10 +621,10 @@ def ts_project_macro(
         include = ["**/*.ts", "**/*.tsx"]
         exclude = []
         if allow_js == True:
-            include.append("**/*.js", "**/*.jsx")
+            include.extend(["**/*.js", "**/*.jsx"])
         if resolve_json_module == True:
             include.append("**/*.json")
-            exclude.append("**/tsconfig*.json")
+            exclude.extend(["**/package*.json", "**/tsconfig*.json"])
```

Comment:
Might not `package*` be too broad? What about listing `package.json` and `package-lock.json` separately here?

Reply:
You're probably right. I've made this update.